components/table/Table.tsx

```diff
@@ -509,7 +509,7 @@ export default class Table<T> extends React.Component<TableProps<T>, any> {
         changeableRowKeys.forEach(key => {
           if (selectedRowKeys.indexOf(key) < 0) {
             selectedRowKeys.push(key);
-          }else {
+          } else {
```

Comment:
That should not changed. It had been fixed in antd-3.0.

https://github.com/ant-design/ant-design/commit/86da4a95ce19c26fc2e7db837498cda94e7cd890